### PR TITLE
Confidential AI doc and some adjustments on doc layout

### PR DESCRIPTION
**Pull request by tolak**
-

### CODE REVIEW

**Comment by HashWarlock** [comment]
Should we call the section Confidential AI and drop the "Reference"

File: confidential-ai/confidential-AI-API.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+# Confidential AI APIs
```
Comment (HashWarlock) [inline confidential-ai/confidential-AI-API.md:1]: Right now this page is not rendering in the preview

File: overview/ai-agent-contract.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# ⚡ AI Agent Contract
```
Comment (HashWarlock) [inline overview/ai-agent-contract.md:1]: This page is not rendering in preview as well. Must be added to `Summary.md`

File: SUMMARY.md [hunk 3]
```diff
@@ -7,7 +7,6 @@
 ## Overview
 
 * [⚖️ Phala Network](overview/phala-network.md)
```
Comment (HashWarlock) [inline SUMMARY.md:9]:
```suggestion
* [⚖️ Phala Network](overview/phala-network.md)
   * [⚡AI Agent Contract](overview/ai-agent-contract.md)
   * [⚡ Confidential AI Inference](overview/confidential-ai-inference.md)
```

File: SUMMARY.md [hunk 4]
```diff
@@ -27,6 +26,14 @@
     * [Khala Governance](overview/pha-token/governance/khala-governance.md)
     * [Setting Up an Account Identity](overview/pha-token/governance/setup-account-identity.md)
 
+## Confidential AI Reference
+
+* [👩‍💻 Getting Started](confidential-ai/getting-started.md)
+* [💎 Confidential AI API](confidential-ai/api.md)
```
Comment (HashWarlock) [inline SUMMARY.md]:
```suggestion
* [💎 Confidential AI API](confidential-ai/confidential-AI-API.md)
```

File: SUMMARY.md [hunk 5]
```diff
@@ -27,6 +26,14 @@
     * [Khala Governance](overview/pha-token/governance/khala-governance.md)
     * [Setting Up an Account Identity](overview/pha-token/governance/setup-account-identity.md)
 
+## Confidential AI Reference
```
Comment (HashWarlock) [inline SUMMARY.md]:
```suggestion
## Confidential AI
```

Comment (tolak) [reply SUMMARY.md]: Should be "Inference"

File: SUMMARY.md (hunk 4 above)
Comment (tolak) [reply SUMMARY.md]: fixed

File: SUMMARY.md (hunk 3 above)
Comment (tolak) [reply SUMMARY.md:9]: Fixed

File: overview/ai-agent-contract.md (hunk 2 above)
Comment (tolak) [reply overview/ai-agent-contract.md:1]: Fixed as your suggestions

File: confidential-ai/confidential-AI-API.md (hunk 1 above)
Comment (tolak) [reply confidential-ai/confidential-AI-API.md:1]: Fixed

File: SUMMARY.md (hunk 5 above)
Comment (tolak) [reply SUMMARY.md]: Explicitly saying Inference will make customers more clear that we only do AI reference in TEE, bc we do not support training.
"Reference" is a typo here, my bad

**Comment by HashWarlock** [comment]
LGTM

**Pull request merged**